docs/conceptual/gpu-arch/mi200-performance-counters.md

```diff
@@ -1,455 +1,591 @@
+| `CPF_CMP_UTCL1_STALL_ON_TRANSLATION` | Cycles | Number of cycles one of the compute UTCL1s is stalled waiting on translation |
+| `CPF_CPF_STAT_BUSY` | Cycles | Number of cycles CPF is busy |
+| `CPF_CPF_STAT_IDLE[∗]` | Cycles | Number of cycles CPF is idle |
+| `CPF_CPF_STAT_STALL` | Cycles | Number of cycles CPF is stalled |
+| `CPF_CPF_TCIU_BUSY` | Cycles | Number of cycles CPF TCIU interface is busy |
+| `CPF_CPF_TCIU_IDLE` | Cycles | Number of cycles CPF TCIU interface is idle |
+| `CPF_CPF_TCIU_STALL[∗]` | Cycles | Number of cycles CPF TCIU interface is stalled waiting on free tags |
 
 #### CPC
 
 ##### CPC counters
 
-| Hardware Counter                 | Unit   | Definition                                          |
-| ---------------------------------| -------| --------------------------------------------------- |
-| `cpc_me1_busy_for_packet_decode` | Cycles | CPC ME1 busy decoding packets                       |
-| `cpc_utcl1_stall_on_translation` | Cycles | One of the UTCL1s is stalled waiting on translation |
-| `cpc_cpc_stat_busy`              | Cycles | CPC busy                                            |
-| `cpc_cpc_stat_idle`              | Cycles | CPC idle                                            |
-| `cpc_cpc_stat_stall`             | Cycles | CPC stalled                                         |
-| `cpc_cpc_tciu_busy`              | Cycles | CPC TCIU interface busy                             |
-| `cpc_cpc_tciu_idle`              | Cycles | CPC TCIU interface idle                             |
-| `cpc_cpc_utcl2iu_busy`           | Cycles | CPC UTCL2 interface busy                            |
-| `cpc_cpc_utcl2iu_idle`           | Cycles | CPC UTCL2 interface idle                            |
-| `cpc_cpc_utcl2iu_stall[∗]`       | Cycles | CPC UTCL2 interface stalled waiting                 |
-| `cpc_me1_dci0_spi_busy`          | Cycles | CPC ME1 Processor busy                              |
+| Hardware counter | Unit   | Definition |
+| :----------------------| :-------| :------------------------ |
+| `CPC_ME1_BUSY_FOR_PACKET_DECODE` | Cycles | Number of cycles CPC ME1 is busy decoding packets |
+| `CPC_UTCL1_STALL_ON_TRANSLATION` | Cycles | Number of cycles one of the UTCL1s is stalled waiting on translation |
+| `CPC_CPC_STAT_BUSY` | Cycles | Number of cycles CPC is busy |
+| `CPC_CPC_STAT_IDLE` | Cycles | Number of cycles CPC is idle |
+| `CPC_CPC_STAT_STALL` | Cycles | Number of cycles CPC is stalled |
+| `CPC_CPC_TCIU_BUSY` | Cycles | Number of cycles CPC TCIU interface is busy |
+| `CPC_CPC_TCIU_IDLE` | Cycles | Number of cycles CPC TCIU interface is idle |
+| `CPC_CPC_UTCL2IU_BUSY` | Cycles | Number of cycles CPC UTCL2 interface is busy |
+| `CPC_CPC_UTCL2IU_IDLE` | Cycles | Number of cycles CPC UTCL2 interface is idle |
+| `CPC_CPC_UTCL2IU_STALL` | Cycles | Number of cycles CPC UTCL2 interface is stalled |
+| `CPC_ME1_DC0_SPI_BUSY` | Cycles | Number of cycles CPC ME1 processor is busy |
 
 ### SPI
```

Comment:
@SwRaw this is an empty heading; We should add content here or remove the following heading (SPI counters).